### PR TITLE
[14.0][ADD] mrp_flattened_bom_xlsx_direct_labour_cost

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**
Depends on https://github.com/OCA/manufacture-reporting/pull/99

### CODE REVIEW

**Review by CasVissers-360ERP — APPROVED** [review]
Does the trick, thnx!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
Dependency already merged.

/ocabot merge nobump

File: mrp_flattened_bom_xlsx_labour_cost/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Export Flattened BOM to Excel with labour cost",
+    "version": "14.0.1.0.0",
+    "category": "Manufacturing",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture-reporting",
+    "license": "AGPL-3",
+    "depends": ["mrp_flattened_bom_xlsx_direct_materials_cost", "mrp_workorder"],
```
Comment (LoisRForgeFlow) [inline mrp_flattened_bom_xlsx_labour_cost/__manifest__.py]: @BernatPForgeFlow I just realized, you are depending on an enterprise module, is this dependency strictly needed?

Comment (BernatPForgeFlow) [reply mrp_flattened_bom_xlsx_labour_cost/__manifest__.py]: No, this is an error. It should depend on the _mrp_ standard module. I change it now.

**Pull request merged**